### ISSUE TITLE
Update identifications without changing ID

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ArctosDB/arctos#614]
```
Email request from Jon Dunnum - came up in discussion of citations and 
identifications 6 Apr 2013:

Can we have the ability to update identifications without changing the ID.  
Want to be able to pull up a bunch of records that are different species but 
ones that I have examined and agree with the field id.  I just want to show 
that they have been reexamined and verified (update them to student or expert 
id, add determined date and agent but no change in id).  Right now you can only 
do a group of the same species because you have to add a species name.
```

Original issue reported on code.google.com by `carla...@gmail.com` on 8 Apr 2013 at 7:40

**Comment by dustymc** [comment]
Needs discussion.

DLM's, and very possibly ONLY DLM's, opinion: There's no "agrees_with_that_one_agent_id" for a reason. The current ID is probably generated using some completely different method - they think it's a moose because the genbank sequence lined up with their genetic idea of mooseness or something. You think it's a moose because it's got moose-antlers. Those things are independent assertions, and they each add their own kind of value to the specimen - especially important in the event of eg, the species being split. Even lacking that precision in the data, having both "looks like a moose to me" (eg, field ID) and "that is definitely a moose" (eg, "expert" ID) can do nothing but add value to the specimen, and I can't understand why there seems to exist a widespread desire to discard them.

Disk space is cheap, stop being so stingy about adding IDs!

We can talk about HOW to add IDs after we've settled what/when. (Could you do what you want if you could upload a spreadsheet, or ..... ?) 

This is perhaps best addressed as part of #515

**Comment by dustymc** [comment]
This was fixed with #811
